scanner.py

```diff
@@ -324,6 +325,7 @@ def cmd_SCANNER_TOUCH(self, gcmd):
         manual_z_offset = gcmd.get_float("Z_OFFSET", self.scanner_touch_config['z_offset'], minval=0)
         test_threshold = gcmd.get_int("THRESHOLD", self.detect_threshold_z, minval=100)
         verbose = gcmd.get_int("DEBUG", 0)
+        randomize = gcmd.get_float("MOVE", 0, maxval=10)
```

Comment:
Should we ensure a minimum value of 0?
Is there a reason for max to be set to 10?

Reply:
I decided on 10 as a safe bet as to not allow users wild values that could drive them over the edges of the bed. This just gave a decent enough square around the centre of the bed to not be always in the same location. Thoughts?